src/sass/components/_products.scss

```diff
@@ -0,0 +1,57 @@
+// .products {
+//   display: flex;
+// }
+
+.products-head {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 15px;
+  &__text {
+    margin-top: 121px;
+    text-transform: uppercase;
+  }
+  &__title {
+    margin-bottom: 144px;
+    text-transform: uppercase;
+  }
+}
+
+.products-grid {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+  gap: 130px;
+  margin-bottom: 98px;
+}
+
+.product-box {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  border-radius: 24px;
+  margin: 0px 20px;
+  &--bg-color:first-child {
+    background-color: #ffb8ca;
+  }
+  &:nth-child(2) {
+    background-color: #f0d1a5;
+  }
+  &:last-child {
+    background-color: #c2e297;
+  }
+  &__title {
+    text-transform: uppercase;
+    margin-top: 161px;
+    margin-bottom: 66px;
+  }
+  &__text {
+    text-align: center;
+    margin-bottom: 30px;
+  }
+  &__link {
+    margin-bottom: 41px;
```

Comment:
nie powinniśmy odsuwać elementu od rodzica marginem tylko paddingiem i najlepiej niech robi to rodzić, w tym wypadku li